src/data/initializers/webData.ts

```diff
@@ -14,6 +14,21 @@ export default function init(
         access_level INTEGER NOT NULL
     )`).run()
 
+    // create default users
+    try {
+        if (exists) {
+            database.prepare(`UPDATE users
+            SET access_level = 2 
+            WHERE email = 'clindsey4@murraystate.edu'`).run()
+            database.prepare(`UPDATE users 
+            SET access_level = 2 
+            WHERE email = 'awatkins23@murraystate.edu'`).run()
+        } else {
+            database.prepare(`INSERT INTO users (email, given_name, family_name, picture, access_level) VALUES ("clindsey4@murraystate.edu", "", "", "", 2)`).run()
```

Comment:
Why not fill in your given name, family name? Also, we should create a developer account to use with unrestricted access. For QA/Testing purposes

Reply:
Yeah, this was temporary while I was struggling to host the website yesterday. I'll add something more secure & less hardcoded in the future.